activestorage/test/models/variant_test.rb

```diff
@@ -6,7 +6,7 @@
 class ActiveStorage::VariantTest < ActiveSupport::TestCase
   test "resized variation of JPEG blob" do
     blob = create_file_blob(filename: "racecar.jpg")
-    variant = blob.variant(resize: "100x100").processed
+    variant = blob.variant(resize_to_fit: [100, 100]).processed
```

Comment:
Since you are maintainer of `image_processing` gem, is there a way to add `resize` in the gem.
If I understood well, inside `image_processing` gem `resize_to_fit` is `resize` from IM...just proposing this so there is no need to change all `resize` calls to `resize_to_fit` inside existing apps 😀

Reply:
ImageProessing gem still supports all ImageMagick operations, the changes in tests and documentation are just to show/test the macros. Any operation sent that is not a macro (`#resize_to_fit`, `#resize_to_limit` etc.) is interpreted to be an ImageMagick operation (via `#method_missing`).

So, with this change you still use `:resize` and any other ImageMagick operations, but you can also additionally use `:resize_to_fit`, `:resize_to_limit` and others.

Reply:
Ahh, that makes sense 👍 ....not sure tho that we need change in documentations/guides then, for me
`resize "100x100"` feels nicer than `resize_to_fit [100, 100]`, but this is just mine preference 😀

Reply:
Yeah, in this particular case `resize "100x100"` is probably nicer. The difference shows in `resize_to_limit [100, 100]` which is `resize "100x100>"` (I find the `>` a bit cryptic), or `resize_to_fill` and `resize_and_pad` which are macros of multiple ImageMagick operations.

While we're here, we should probably recommend using `resize "100x100>"` (or `#resize_to_limit`), as when you physically enlarge an image that is smaller than specified dimensions, it will generally look worse than if you just display the original image enlarged. That of course won't happen with thumbnails as small as `100x100`, but it could with larger thumbnails.

What do you think?

Reply:
Actually, then people would have to remember to add the `width`/`height` HTML attributes to the `<img>` tags. I'll just revert to using `resize`, and additionally show an example of using a `resize_*` macro.